script/tool/test/native_test_command_test.dart

```diff
@@ -895,9 +898,13 @@ void main() {
         final Directory pluginExampleDirectory =
             pluginDirectory1.childDirectory('example');
 
-        processRunner.processToReturn = MockProcess.succeeding();
         // Simulate a project with unit tests but no integration tests...
-        processRunner.resultStdout = '{"project":{"targets":["RunnerTests"]}}';
+        const String projectListOutput =
+            '{"project":{"targets":["RunnerTests"]}}';
```

Comment:
nit: Doesn't Dart have a function that can turn Dart data structures to json strings?

Reply:
Done.